datasets.py

```diff
@@ -919,3 +920,110 @@ def from_path(self, file_path):
         anns = [image_anns, gender_info, [country], file_path, scene_group]
 
         return image, anns
+
+class CityScapes(data.Dataset):
```

Comment:
I think all datasets so far names like "CoCoDataset", "[blank]Dataset", etc. so for consistency can you rename this CityScapesDataset

Reply:
Done!